src/views/v2/emails/template-parts/body/footer.php

```diff
@@ -13,11 +13,22 @@
  *
  * @since 5.5.9
  *
- * @var Tribe_Template  $this  Current template object.
- * @var string $footer_content    HTML of footer content.
- * @var bool   $footer_credit     Show the footer credit?
- * @var string $header_bg_color   Header background color.
- * @var string $header_text_color Header text color.
+ * @var Tribe_Template $this               Current template object.
+ * @var Email_Abstract $email              The email object.
```

Comment:
```suggestion
 * @var \TEC\Tickets\Emails\Email_Abstract $email              The email object.
```

Let's doa bulk replace on this one as well.